docker/entrypoint.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+# Define the start command
+START_COMMAND="/usr/local/bin/sim-cli"
+
+# Check for a custom configuration path
+if [[ ! -z ${CONFIG_PATH} ]]; then
+    START_COMMAND="$START_COMMAND --config $CONFIG_PATH"
```

Comment:
Needs rebase, we removed `--config` recently so now it's just `sim-cli sim.json` (since it's always required). 
Also decided to refer to that file as `sim.json` since we may one day want to have a config file for the _actual_ simulation (containing things like log level and print batch etc).